pages/give-devices.js

```diff
@@ -9,21 +9,19 @@ import SelectFormControl from '../components/form/SelectFormControl';
 import TextareaFormControl from '../components/form/TextareaFormControl';
 
 // Use ISO 3166 country codes per https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes
-const COUNTRIES = [
-  { value: 'US', label: 'United States' },
-  { value: 'CA', label: 'Canada' },
-  { value: 'MX', label: 'Mexico' },
-  { value: 'DE', label: 'Germany' },
-  { value: 'LV', label: 'Latvia' },
-  { value: 'IN', label: 'India' },
-  { value: 'BD', label: 'Bangladesh' },
-  { value: 'PH', label: 'Philippines' },
-];
+const countryList = require('country-list');
+
+const countries = countryList.getData();
+countries.sort((a, b) => (a.name > b.name && 1) || -1);
+const COUNTRIES = [];
+countries.forEach((element) => {
+  COUNTRIES.push({ value: element.code, label: element.name });
+});
```

Comment:
Rather than `getData`, `getNames` appears more appropriate as we do not seem to care for the country codes. The following solution helps to prevent duplicate storage and ensures true following of the use of `const` and the formatting pattern.

```suggestion
const COUNTRIES = countryList
  .getNames()
  .sort((a, b) => (a > b && 1) || -1)
  .map((value) => {
    return { label: value };
  });
```

Reply:
@RedFox0x20 please skip the `return` :) This should be enough:
```javascript
 .map((value) => ({ label: value });
```

Reply:
One more thing - why can't we just use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare for string comparison?